configs/reconciler-unittest.yaml

```diff
@@ -6,7 +6,7 @@ db:
   blockQueries: true
   logQueries: true
   postgres:
-    host: "localhost"
+    host: "127.0.0.1"
```

Comment:
just by curiosity, normally these two values should be synonym, DNS will handle the look up from localhost to IP 127.0.0.1 correctly, do we have a specific reason to change?

Reply:
It seems like, we have an IPv6 problem here, that some times localhost will not be properly translated to 127.0.0.1

Reply:
aha, good to know, thanks!